src_types/public/index.d.ts

```diff
@@ -1,4 +1,4 @@
 /// <reference path="../common/anyProperties.d.ts" />
 /// <reference path="../common/emptyBuffer.d.ts" />
-/// <reference path="../common/void$w$w.d.ts" />
+/// <reference path="../common/anyString$wSelector.d.ts" />
```

Comment:
1. allowing any string is just for the "public" context, so i would move it to the "public" folder (and probably just call it $w, same as for pages).
2. i think you still need the `void$w$w.d.ts` for backend....